app/assets/stylesheets/forever_style_guide/modules/_modal.scss

```diff
@@ -0,0 +1,100 @@
+$modal-padding-footer: 15px;
+$modal-form-width-lg: 355px;
+$modal-product-suggestion-sm: 170px;
+
+.modal-close-position {
+  float: right;
+}
+
+.modal-theme {
+  color: $color-white;
+}
+
+.modal-title {
+  color: inherit;
+  font-family: $font-face-light;
+}
+
+.modal-footer-buttons {
+  padding: $modal-padding-footer;
+  border-top: 1px solid $color-gray-400;
+  text-align: right;
+}
+
+.modal-footer-txt {
+  font-family: $font-face-bold;
+  color: $color-gray-600;
+  line-height: 3;
+}
+
+.modal-footer-info {
+  padding: 10px;
+}
+
+.modal-body-info {
+  padding: 30px;
+}
+
+.modal-product-name-sm {
+  margin: 10px auto 0;
+  max-width: $modal-product-suggestion-sm;
+  color: $color-gray-600;
+  font-family: $font-face-bold;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+}
+
+.modal-footer-products {
+  padding: $modal-padding-footer;
+}
+
+.product-preview:hover .modal-product-name-sm {
+  color: $color-secondary;
+}
+
+.close {
+  float: right;
+}
+
+.modal-footer-txt.checkbox {
+  line-height: 1.5;
+}
+
+.media.text-center {
+  padding: 0px 40px;
+  margin-top: $modal-padding-footer;
+}
+
+.modal-content.modal-promo {
+  background-image: url('modals/promobg.jpg');
+  background-size: cover;
+  height: 475px;
+}
+
+.modal-promo-footer,
+.modal-promo-header {
+  border: none;
+}
+
+.modal-promo-content {
+  margin-top: 165px;
+  padding: 0px 50px;
+  color: $color-gray-600;
+  font-size: 25px;
```

Comment:
It's preferable to use a standard font-size. If it makes more sense to use a custom value, i'd recommend including it as a variable or adding a comment.